zap/src/main/java/org/parosproxy/paros/control/Control.java

```diff
@@ -335,22 +335,17 @@ public void run() {
     }
 
     /**
-     * Sets the non zero value ZAP will exit cleanly with. ZAP may still exit with a non zero value
-     * if a serious error occurs. This will work however ZAP is run but it makes more sense if ZAP
-     * is run in cmdline mode. Any add-on can set an exit status so attempts to reset the status to
-     * zero will be rejected.
+     * Sets the value ZAP will exit cleanly with. ZAP may still exit with a non zero value if a
+     * serious error occurs. This will work however ZAP is run but it makes more sense if ZAP is run
+     * in cmdline mode.
```

Comment:
```suggestion
     * Sets the value ZAP will exit cleanly with. ZAP may still exit with a non-zero value if a
     * serious error occurs. This will work however ZAP is run but it makes more sense if ZAP is run
     * in cmdline mode.
```